### PR TITLE
[zh-cn]: update translation of break property statement

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/statements/break/index.md [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+
+执行到 `break label;` 时，程序会跳出标记为 `label` 的语句，并继续执行其后的下一条语句。`break` 语句需要嵌套在引用的标签内。标记语句可以是任何语句（通常是 {{jsxref("Statements/block", "block", "", 1)}} 语句），它不一定是另一个循环语句。
 
-break 语句不能在 function 函数体中直接使用，break 语句应嵌套在要中断的当前循环、switch 或 label 语句中。
+即使函数或类进一步嵌套在循环中，`break` 语句（无论是否带有标签）也不能在脚本、模块、函数体或[静态初始化块](/zh-CN/docs/Web/JavaScript/Reference/Classes/Static_initialization_blocks)的顶层使用。
 
 ## 示例
 
-### break in while loop
+### 中止 while 循环
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/break/index.md:34]:
```suggestion
### 中断 while 循环
```

File: files/zh-cn/web/javascript/reference/statements/break/index.md [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
 
-下面的函数里有个 `break` 语句，当 `i` 为 3 时，会中止 {{jsxref("Statements/while", "while")}} 循环，然后返回 3 \* `x` 的值。
+以下函数有一个 `break` 语句，当 `i` 等于 3 时中止 {{jsxref("Statements/while", "while")}} 循环，然后返回 `3 * x` 的值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/break/index.md:36]:
```suggestion
以下函数有一个 `break` 语句，当 `i` 等于 3 时终止 {{jsxref("Statements/while", "while")}} 循环，然后返回值 `3 * x`。
```

File: files/zh-cn/web/javascript/reference/statements/break/index.md [hunk 3]
```diff
@@ -43,85 +50,82 @@ function testBreak(x) {
...
-  console.log("2"); // skipped
+  console.log("2"); // 跳过这一行
 }
 ```
 
-### break in labeled blocks that throw
+### 非法 break 语句
 
-下面的代码同样使用了 `break` 语句和被标记的块语句，但是产生了一个语法错误，因为它的 `break` 语句在 `block_1` 中，但是引用了 `block_2`。`break` 语句必须内嵌在它引用的标签中。
+以下代码也使用了带 `break` 语句的标签，但是会生成语法错误，因为其 `break` 语句引用了 `block2`，但它并未嵌套在 `block2` 内。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/break/index.md:90]:
```suggestion
一个 `break` 语句必须嵌套在它引用的任何标签内部。以下代码也使用了带 `break` 语句的标签，但是会产生语法错误，因为其 `break` 语句引用了 `block2`，但它并未嵌套在 `block2` 内。
```

File: files/zh-cn/web/javascript/reference/statements/break/index.md [hunk 4]
```diff
@@ -43,85 +50,82 @@ function testBreak(x) {
...
-  console.log ('2');
+block2: {
+  console.log("2");
 }
 ```
 
-### break within functions
-
-在下面的代码同样会产生 SyntaxError，因为它并没被正确的使用在循环、switch 或 label 语句中。
+以下代码示例中，在嵌套在循环或带标签块中的函数内使用 `break` 语句也会生成语法错误，而 `break` 语句旨在跳出这些循环或带标签块。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/break/index.md:103]:
```suggestion
以下代码示例中，在嵌套在循环或带标签块中的函数内使用 `break` 语句也会产生语法错误，而 `break` 语句旨在跳出这些循环或带标签块。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/statements/break/index.md [hunk 5]
```diff
@@ -1,39 +1,46 @@
...
 - `label` {{optional_inline}}
-  - : 与语句标签相关联的标识符。如果 break 语句不在一个循环或 {{jsxref("Statements/switch", "switch")}} 语句中，则该项是必须的。
+  - : 与要中断的语句的标签关联的标识符。如果 `break` 语句未嵌套在循环或 {{jsxref("Statements/switch", "switch")}} 中，则需要标签标识符。
 
 ## 描述
 
-`break`语句包含一个可选的标签，可允许程序摆脱一个被标记的语句。`break`语句需要内嵌在引用的标签中。被标记的语句可以是任何 {{jsxref("Statements/block", "块")}}语句；不一定是循环语句。
+执行到 `break;` 时，程序会跳出最内层的 `switch` 或[循环](/zh-CN/docs/Web/JavaScript/Reference/Statements#迭代)语句，并继续执行其后的下一条语句。
+
+执行到 `break label;` 时，程序会跳出标记为 `label` 的语句，并继续执行其后的下一条语句。`break` 语句需要嵌套在引用的标签内。标记语句可以是任何语句（通常是 {{jsxref("Statements/block", "block", "", 1)}} 语句），它不一定是另一个循环语句。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/break/index.md:28]:
```suggestion
执行到 `break label;` 时，程序会跳出标记为 `label` 的语句，并继续执行其后的下一条语句。`break` 语句需要嵌套在引用的标签内。标记语句可以是任何语句（通常是{{jsxref("Statements/block", "块", "", 1)}}语句），它不一定是另一个循环语句。
```

**Review by yin1999 — COMMENTED** [review]
可以翻译一下：

**Review by JasonLamv-t — APPROVED** [review]
Well done! @T34-active

**Comment by T34-active** [comment]
> Well done! @T34-active

Thanks for the suggestion.

**Pull request merged**